src/libraries/Microsoft.PowerFx.Connectors/Public/ConnectorType.cs

```diff
@@ -329,5 +331,35 @@ private Dictionary<string, FormulaValue> GetEnum()
 
             return enumDisplayNames.Zip(enumValues, (dn, ev) => new KeyValuePair<string, FormulaValue>(dn, ev)).ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
         }
+
+        public bool Equals(ConnectorType other)
+        {
+            if (other == null)
+            {
+                return false;
+            }
+
+            if (ReferenceEquals(this, other))
+            {
+                return true;
+            }
+
+            return GetHashCode() == other.GetHashCode();            
```

Comment:
> return GetHashCode() == other.GetHashCode() [](http://example.com/codeflow?start=11&length=44)

how is this used?
Having same HashCode doesn't necessarily mean equal objects.

Reply:
it's a bug I need to fix!